generic_client.go

```diff
@@ -5,28 +5,31 @@ import (
 	"encoding/json"
 	"fmt"
 	"io"
-	"io/ioutil"
 	"net/http"
 	"net/url"
 	"regexp"
 	"strings"
+	"time"
+
+	"github.com/opentable/sous/util/logging"
+	"github.com/opentable/sous/util/logging/messages"
```

Comment:
I can see we're using this import for more than interfaces, if we keep doing that I guess above comment is moot.